REFERENCE.md

```diff
@@ -0,0 +1,4514 @@
+
+Default value: `undef`
+
+##### `keepalive`
+
+Data type: `Optional[Enum['on', 'off']]`
+
+Determines whether to enable persistent HTTP connections with the `KeepAlive` directive
+for the virtual host. By default, the global, server-wide `KeepAlive` setting is in effect.<br />
+Use the `keepalive_timeout` and `max_keepalive_requests` parameters to set relevant options
+for the virtual host.
+
+Default value: `undef`
+
+##### `keepalive_timeout`
+
+Data type: `Any`
+
+Sets the `KeepAliveTimeout` directive for the virtual host, which determines the amount
+of time to wait for subsequent requests on a persistent HTTP connection. By default, the
+global, server-wide `KeepAlive` setting is in effect.<br />
+This parameter is only relevant if either the global, server-wide `keepalive` parameter or
+the per-vhost `keepalive` parameter is enabled.
+
+Default value: `undef`
+
+##### `max_keepalive_requests`
+
+Data type: `Any`
+
+Limits the number of requests allowed per connection to the virtual host. By default,
+the global, server-wide `KeepAlive` setting is in effect.<br />
+This parameter is only relevant if either the global, server-wide `keepalive` parameter or
+the per-vhost `keepalive` parameter is enabled.
+
+Default value: `undef`
+
+##### `auth_kerb`
+
+Data type: `Boolean`
+
+Enable `mod_auth_kerb` parameters for a virtual host.<br />
+Usage typically looks like:
+``` puppet
+apache::vhost { 'sample.example.net':
+  auth_kerb              => `true`,
+  krb_method_negotiate   => 'on',
+  krb_auth_realms        => ['EXAMPLE.ORG'],
+  krb_local_user_mapping => 'on',
+  directories            => {
+    path         => '/var/www/html',
+    auth_name    => 'Kerberos Login',
+    auth_type    => 'Kerberos',
+    auth_require => 'valid-user',
+  },
+}
+```
+
+Default value: `false`
+
+##### `krb_method_negotiate`
+
+Data type: `Any`
+
+Determines whether to use the Negotiate method.
+
+Default value: 'on'
+
+##### `krb_method_k5passwd`
+
+Data type: `Any`
+
+Determines whether to use password-based authentication for Kerberos v5.
+
+Default value: 'on'
+
+##### `krb_authoritative`
+
+Data type: `Any`
+
+If set to `off`, authentication controls can be passed on to another module.
+
+Default value: 'on'
+
+##### `krb_auth_realms`
+
+Data type: `Any`
+
+Specifies an array of Kerberos realms to use for authentication.
+
+Default value: []
+
+##### `krb_5keytab`
+
+Data type: `Any`
+
+Specifies the Kerberos v5 keytab file's location.
+
+Default value: `undef`
+
+##### `krb_local_user_mapping`
+
+Data type: `Any`
+
+Strips @REALM from usernames for further use.
+
+Default value: `undef`
+
+##### `krb_verify_kdc`
+
+Data type: `Any`
+
+This option can be used to disable the verification tickets against local keytab to prevent
+KDC spoofing attacks.
+
+Default value: 'on'
+
+##### `krb_servicename`
+
+Data type: `Any`
+
+Specifies the service name that will be used by Apache for authentication. Corresponding
+key of this name must be st
```

Comment:
Type: "ammount" and "allow".  "...maximum amount of requests that *are* allowed to queue whenever..."

Reply:
updated